package.json

```diff
@@ -259,6 +263,11 @@
 					"command": "gerrit.openOriginal",
 					"when": "view == gerrit:changeExplorer && viewItem =~ /filechange/ && viewItem =~ /modified/",
 					"group": "open@3"
+				},
+				{
+					"command": "gerrit.checkoutBranch",
```

Comment:
Can you also add an entry to `contributes.menus.commandPalette` in this file with `"when": "false"`. That ensures that this command doesn't show up when you do `ctrl+shift+p`. If it were to show up and someone used it, your command handler would fail because no change would be passed to it.